src/ol_infrastructure/applications/codejail/Pulumi.applications.codejail.xpro.Production.yaml

```diff
@@ -4,8 +4,8 @@ encryptedkey: AQICAHg/+QzF9hGIaoayDitgnEVHEhuaANONVQQOnqpkIsol1gFIp45VyYwQaGk+qW
 config:
   aws:region: us-east-1
   codejail:auto_scale:
-    desired: 5
-    max: 10
+    desired: 2
```

Comment:
Same minimum of 3 here. Apparently xpro has a significant usage of codejail in their courses.